Add seconds display animation

Tangsang2003 opened the pull request:
This pull request resolves Issue #1 by animating the seconds display. The seconds now update dynamically in real-time.
## Changes Made:
- Added `setInterval(updateDateTime, 1000)` instead of simple one-time update during page load.

## Suggestions:
- The dependency on the API call can introduce some delays due to Network latency, which can impact the seconds update.
- To fix that, we can ensure server sync every 5 minutes. Please open a new issue, if you want me to add that.

## Closes:
- Issue #1

Tangsang2003 committed:
Add seconds display animation

njsubedi commented:
Let’s not hit the API each second. Only hit the API once, then calculate the seconds locally.

Tangsang2003 committed:
Update the code to fetch time from API only once during initial load

Tangsang2003 commented:
Hi, I have updated the code to hit the API only once and then calculate the seconds locally.
I have removed the getNTPTime function with fetchServerTime function.
Also, I have added a formatDateTime function to match the format in the original site.
Please review the changes and let me know if I need to add any more changes.

njsubedi commented:
Merged and it’s auto-deployed to nepalidate.org
